src/redux/reducers/experimentSettings/index.js

```diff
@@ -34,6 +40,15 @@ const experimentSettingsReducer = (state = initialState, action) => {
     case EXPERIMENT_SETTINGS_PIPELINE_STATUS_ERROR: {
       return pipelineStatusError(state, action);
     }
+    case HYDRATE: {
+      const experimentInfo = action.payload.experimentSettings.info;
+
+      if (experimentInfo) {
+        return { ...state, info: experimentInfo };
+      }
+
+      return state;
+    }
```

Comment:
Hydration logic. `HYDRATE` is called when the server tries to merge in changes to the Redux store that happened before the client loaded the page. We check if experiment information was changed from the default, and if so, we merge it into the store.